UpdateLogic.hs

```diff
@@ -199,7 +199,13 @@ getInstallersResults keys instP daedalusRev destDir = do
 
 handleCIResults :: InstallerPredicate -> Turtle.FilePath -> Either Text [CIResult] -> IO [CIResult2]
 handleCIResults instP destDir (Right rs) = do
+  print rs
   let rs' = filter instP rs
+  mapM_ (\result -> do
+    print result
+    print $ (installerNetwork . ciResultFilename) result
+    ) rs
+  print rs'
   fetchCIResults destDir rs'
```

Comment:
do we still need the print's here?

Reply:
Will leave them for now as it helps with debugging and clears from the screen quickly after they print (but can still be retrieved if needed).